tests/ops/op_math/test_exp.py

```diff
@@ -546,7 +546,7 @@ def test_trotter_is_used_if_num_steps_is_defined(self):
         ]
         assert len(dec) == len(expected_decomp)
         for op1, op2 in zip(dec, expected_decomp):
-            qml.equal(op1, op2)
+            qml.assert_equal(op1, op2)
```

Comment:
🤦

Reply:
such a useful test.

Reply:
I opened a separate PR to fix this test: https://github.com/PennyLaneAI/pennylane/pull/5867